### PR TITLE
Lecture/fernando gomez react redux

### PR DESCRIPTION
**Pull request by shd8**

### CODE REVIEW

File: fernando-gomez/lectures/toh-redux/src/App.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
+.App {
```
Comment (gilbecao) [inline fernando-gomez/lectures/toh-redux/src/App.css:1]: Elimina los estilos que no vas a utilizar

**Comment by gilbecao** [comment]
Muy bien los commits detallados 👏

File: fernando-gomez/lectures/toh-redux/src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      <Provider store={store}>
+        <Router>
+          <h1>Tour of heroes</h1>
+          <nav>
+            <Link to="/">Dashboard</Link>
+            <Link to="/heroes/">Heroes</Link>
+          </nav>
+          <Switch>
+            <Route exact path="/" component={Dashboard} />
+            <Route exact path="/detail/:heroId" component={HeroDetail} />
```
Comment (Laura-SM) [inline fernando-gomez/lectures/toh-redux/src/App.jsx:28]: Are you sure all the routes need to be exact?

**Review by Laura-SM — APPROVED** [review]
Good job!

**Review by IrenePerezAlonso — APPROVED** [review]

**Review by IrenePerezAlonso — APPROVED** [review]

**Review by Emerson-feral — APPROVED** [review]

**Review by gaxxrodri — APPROVED** [review]

**Review by CheCabeza — APPROVED** [review]

**Pull request merged**